### PR TITLE
add incremental-dom v0.2.0

### LINKED ISSUE #5729: [Request] incremental-dom
incremental-dom
https://github.com/google/incremental-dom
incremental-dom
Apache 2.0
http://google.github.io/incremental-dom/#about

I put an issue here because it is GOOGLE:

https://github.com/google/incremental-dom/issues/139

Needs to be installed with NPM to get the `dist` files

```
.
├── incremental-dom-cjs.js
├── incremental-dom-cjs.js.map
├── incremental-dom-min.js
├── incremental-dom-min.js.map
├── incremental-dom.js
└── incremental-dom.js.map

```

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/google/incremental-dom
**Watch 72
  Star 1,483
 Fork 46**
@LeaYeh could u help me check this PR for #5729 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/incremental-dom/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "name": "sepandparhami",
+      "email": "sparhami@google.com"
+    }
+  ],
+  "npmName": "incremental-dom",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "incremental-dom*.+(js|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/incremental-dom/package.json]: why not `**/*`?

**Comment by LeaYeh** [comment]
@maruilian11  why you add npm autoupdate config but not git config?

**Comment by maruilian11** [comment]
@LeaYeh i'm not sure if i use git auto-update, i can get `dist` folder.
cc #5729

**Comment by PeterDaveHello** [comment]
npm is good

**Comment by LeaYeh** [comment]
ok, i think this pr is ok, plz rebase it again :)

**Comment by PeterDaveHello** [comment]
@LeaYeh You should ping me after the rebase, not now ...

**Comment by PeterDaveHello** [comment]
@LeaYeh And the pr's title is too long.

**Comment by LeaYeh** [comment]
@PeterDaveHello sorry for that, and what is the max len of title?

**Comment by PeterDaveHello** [comment]
I think, maybe within 50 chars, like `add incremental-dom v0.2.0`

**Comment by maruilian11** [comment]
@LeaYeh is that ok？

**Comment by LeaYeh** [comment]
i think it is fine 
@PeterDaveHello

**Comment by PeterDaveHello** [comment]
should rebase ... @LeaYeh don't you know that or forgot again?

**Comment by maruilian11** [comment]
@LeaYeh i have rebased it ~

**Comment by LeaYeh** [comment]
ok, this pr is ok.
@PeterDaveHello  could you check this pr for me?

**Comment by PeterDaveHello** [comment]
Don't you need to add the manually minified file `incremental-dom-cjs.min.js` into `requiredFiles` array?
And also I wonder if we really need it, maybe it's not for browser, if it's for browser, the official developer may minify it as like incremental-dom-min.js, please confirm that.

**Comment by maruilian11** [comment]
@LeaYeh could you help me check the PR again?
i use `!(incremental-dom-cjs*)` to not include incremental-dom-cjs* in dist

**Comment by maruilian11** [comment]
@Amomo could you help me check this PR? thank you~

**Comment by maruilian11** [comment]
ping @Amomo

**Comment by PeterDaveHello** [comment]
> i use !(incremental-dom-cjs*) to not include incremental-dom-cjs* in dist

Did you confirm that is incremental-dom-cjs for browser or not?

I wonder if this **project** if for browser usage or not ...

**Comment by maruilian11** [comment]
@PeterDaveHello yes, cc https://github.com/google/incremental-dom/issues/186

**Pull request merged**